client/components/OrdersView.js

```diff
@@ -0,0 +1,141 @@
+import React, { useEffect, useState } from 'react'
+import {connect} from 'react-redux'
+import { useParams } from 'react-router-dom'
+import { fetchUser } from '../store/singleUser'
+import CartItemCard from './CartItemCard'
+
+const formatOrderDate = (isoString) => {
+    const date = new Date(isoString)
+    return <time className="order-date" dateTime={isoString}>{date.toTimeString()}</time>
+}
+
+const OrderListLine = ({order}) => (
+    <div className="d-flex justify-content-between align-items-center">
+        <span className="order-id">{order.id}</span>&nbsp;
+        {formatOrderDate(order.updatedAt)}
+        <span className="badge badge-info badge-pill">{order.products.length}</span>
+    </div>
+)
+
+/**
+ * Renders info pane for chosen order
+ */
+const OrderDetails = ({order}) => (
+    <div className="card col">
+        <div className="card-header">
+            Order Details (id #{order.id})
+        </div>
+        <div className="card-body">
+            <h5 className="card-title">{formatOrderDate(order.updatedAt)}</h5>
+            <p className="card-text">...order details go here...</p>
```

Comment:
is this order details go here line supposed to be in here??

Reply:
I didn't get around to laying out the actual order metadata